dissect/target/plugins/apps/ssh/openssh.py

```diff
@@ -64,7 +65,13 @@ class SSHPlugin(Plugin):
     __namespace__ = "ssh"
 
     def check_compatible(self):
-        return len(list(self.target.users())) > 0 and self.target.os != "windows"
+        return len(list(self.target.users())) > 0 or self._sshd_directory().exists()
+
+    def _sshd_directory(self) -> TargetPath:
```

Comment:
Would it be useful to make this an `@property`? And then also possibly caching it by saving it to some instance variable.

Reply:
Even better  :laughing: